### PR TITLE
Add terms of use

### LINKED ISSUE #15: Document API terms of service
- [ ] Add API Terms of Service document describing what users can and cannot do with the API.

### PR DESCRIPTION
**Pull request by hburn7**
Adds a basic terms of use for the API. Not sure what else we'd want to put here. Closes #15

### CODE REVIEW

File: docs/topics/Terms-of-Use.md [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+
+By using the o!TR API, you agree to abide by these terms of use. The o!TR API is provided publicly, for free, for all users by the osu! Tournament Rating Team ("o!TR Team", "us", "our").
+
+## Scraping
+
+The API must not be used to scrape information. Scraping data, such as tournaments, matches, etc. is forbidden and will result in a permanent revocation of API access. Please see the [data](Data.md) page for more information on conveniently importing our dataset.
+
+## Accreditation
+
+Data which is sourced from us should be credited properly. Misleading users to believe data created by us is provided by another source, or failing to credit this information as being created by us, is not allowed.
```
Comment (cytusine0) [inline docs/topics/Terms-of-Use.md:11]: do *we* mention (when relevant) how we source our data?

File: docs/topics/Terms-of-Use.md [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+> 
+{style="warning"}
+
+## Abuse
+
+Users who abuse this service will have their API access revoked. For example, repeatedly encountering 429 errors will result in access revocation.
+
+### Rate Limit Increases
+
+If you wish to receive an increase to the API rate limit of your application, please [contact us](Contact.md) with your use case.
```
Comment (cytusine0) [inline docs/topics/Terms-of-Use.md:23]: is it worth writing here what the default rate limits are?

File: docs/topics/Terms-of-Use.md [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+
+Data which is sourced from us should be credited properly. Misleading users to believe data created by us is provided by another source, or failing to credit this information as being created by us, is not allowed.
+
+> For example, informing users of your application that information is provided by osu! officially is not allowed because it is not true.
+> 
+{style="warning"}
+
+## Abuse
+
+Users who abuse this service will have their API access revoked. For example, repeatedly encountering 429 errors will result in access revocation.
```
Comment (cytusine0) [inline docs/topics/Terms-of-Use.md:19]: what counts as "abuse" besides scraping / exceeding rate limit (which are already mentioned on this page)?

**Review by cytusine0 — COMMENTED** [review]
:eh:

File: docs/topics/Terms-of-Use.md (hunk 1 above)
Comment (hburn7) [reply docs/topics/Terms-of-Use.md:11]: Could probably go under the Data page? Somewhere under the primary section at least.

File: docs/topics/Terms-of-Use.md (hunk 2 above)
Comment (hburn7) [reply docs/topics/Terms-of-Use.md:23]: I think that info is better suited under the actual spec section (coming at some point in the future). That way we don't have to maintain it in two places.

File: docs/topics/Terms-of-Use.md (hunk 3 above)
Comment (hburn7) [reply docs/topics/Terms-of-Use.md:19]: Honestly I don't know

Comment (myssto) [reply docs/topics/Terms-of-Use.md:19]: > repeatedly encountering 429 errors
Avoid jargon where possible. I think it reads a lot better to say "repeatedly exceeding the rate limit" or something to that effect

File: docs/topics/Terms-of-Use.md [hunk 4]
```diff
@@ -0,0 +1,23 @@
+# Terms of Use
+
+By using the o!TR API, you agree to abide by these terms of use. The o!TR API is provided publicly, for free, for all users by the osu! Tournament Rating Team ("o!TR Team", "us", "our").
+
+## Scraping
+
+The API must not be used to scrape information. Scraping data, such as tournaments, matches, etc. is forbidden and will result in a permanent revocation of API access. Please see the [data](Data.md) page for more information on conveniently importing our dataset.
```
Comment (myssto) [inline docs/topics/Terms-of-Use.md:7]: I would say that this counts as a form of abuse and the two of these categories can probably be merged. I would keep the `## Abuse` heading and list scraping as one of the examples of what we consider abuse

File: docs/topics/Terms-of-Use.md (hunk 1 above)
Comment (hburn7) [reply docs/topics/Terms-of-Use.md]: For now I just added one extra sentence about this. Page can come later.

**Review by cytusine0 — APPROVED** [review]
:okeh:

**Pull request merged**